queries/latex/highlights.scm

```diff
@@ -0,0 +1,160 @@
+;; Math
+[
+ (displayed_equation)
+ (inline_formula)
+] @text.math
+
+;; This highlights the whole environment like vimtex does
+((environment
+  (begin
+   name: (word) @_env)) @text.math
+   (#match? @_env "^(displaymath|eqn|eqnarray|align)[*]?$"))
+
+;; This at the begining of the file would be the alternative to highlight
+;; only the interior of the environment
+;((environment
+  ;(begin
+   ;name: (word) @_env) @none
+   ;(end) @none) @text.math
+   ;(#match? @_env "^(displaymath|eqn|eqnarray)[*]?$"))
+
+[
+  (generic_command_name)
+  "\\begin"
+  "\\end"
+  "\\newcommand"
+  "\\renewcommand"
+  "\\DeclareRobustCommand"
+  "\\DeclareMathOperator"
+  "\\newglossaryentry"
+  "\\caption"
+  "\\cite"
+  "\\label"
+  "\\newlabel"
+  "\\label"
+  "\\ref"
+] @function.macro
+
+(comment) @comment
+
+(bracket_group) @parameter
+
+[(math_operator) "="] @operator
+
+[
+  "\\usepackage"
+  "\\documentclass"
+  "\\input"
+  "\\include"
+  "\\subfile"
+  "\\subfileinclude"
+  "\\subfileinclude"
+  "\\includegraphics"
+  "\\addbibresource"
+  "\\bibliography"
+  "\\includesvg"
+  "\\includeinkscape"
+  "\\usepgflibrary"
+  "\\usetikzlibrary"
+] @include
+
+[
+  "\\part"
+  "\\chapter"
+  "\\section"
+  "\\subsection"
+  "\\subsubsection"
+  "\\paragraph"
+  "\\subparagraph"
+] @type
+
+"\\item" @punctuation.special
+
+((word) @punctuation.delimiter
+(#eq? @punctuation.delimiter "&"))
+
+
+(label_definition
+ name: (_) @text.reference)
+(label_reference
+ label: (_) @text.reference)
+(equation_label_reference
+ label: (_) @text.reference)
+(label_reference
+ label: (_) @text.reference)
+(label_number
+ label: (_) @text.reference)
+
+(citation
+ key: (word) @text.reference)
+
+(key_val_pair
+  key: (_) @method ; like JS
+  value: (_))
+
+["(" ")" "[" "]" "{" "}"] @punctuation.bracket
+
+(environment
+  begin: (begin
+          name: (_) @text.environment.name) @text.environment
+  end: (end) @text.environment)
```

Comment:
Indeed, this should also capture `name`.

Reply:
should work now

Reply:
By the way, how do you update a checked-out PR that is being force-pushed to? I keep getting merge conflicts even without local changes...

Reply:
It does not for me?

Reply:
@clason, that would be `git pull --force` in the manual way 😉

Reply:
`fatal: Not possible to fast-forward, aborting.`

Reply:
Ah, right. The name is also at the end tag... `git reset --hard remote/branch`?

Reply:
(probably need to add `--rebase` to it)

Reply:
I'll try `git reset` next. (`gh` is a bit brittle...)

Reply:
Nope, doesn't work either...